devimint/Cargo.toml

```diff
@@ -18,6 +18,7 @@ axum = { workspace = true, features = ["tracing"] }
 bitcoin = { workspace = true }
 bitcoincore-rpc = { workspace = true }
 bon = { workspace = true }
+chrono = { workspace = true }
```

Comment:
Oh dear. Used to be `chrono` was the go to date time library. Then I think it was unmaintained, and `time` become go to. Now I've checked on crates.io and chrono seems maintained again, and they have similiar usage stats. :shrug: 

And we already depend on both.